pkg/kubectl/resource_printer.go

```diff
@@ -1538,7 +1538,8 @@ func printNode(node *api.Node, w io.Writer, options PrintOptions) error {
 	}
 
 	if options.Wide {
-		if _, err := fmt.Fprintf(w, "\t%s", getNodeExternalIP(node)); err != nil {
+		osImage, kernelVersion := node.Status.NodeInfo.OSImage, node.Status.NodeInfo.KernelVersion
+		if _, err := fmt.Fprintf(w, "\t%s\t%s\t%s", osImage, kernelVersion, getNodeExternalIP(node)); err != nil {
```

Comment:
what if these are empty? Should consider some default value, like `<unknown>` or `<none>`